suites/squid/common/sanity/01-rbd.yaml

```diff
@@ -0,0 +1,273 @@
+tests:
```

Comment:
can you please add which `conf` will be using for this test suite?

Reply:
All suites in sanity directory will be using common cluster configuration. Will add readme file once complete all activity.